data/crac-creation/crac-creator-csa-profiles/src/main/java/com/powsybl/openrao/data/craccreation/creator/csaprofile/craccreator/CsaProfileCracCreator.java

```diff
@@ -89,6 +90,17 @@ private void addCsaInstants() {
         // TODO : add other curative instants here
     }
 
+    private void addRaUsageLimits(CracCreationParameters parameters) {
+        parameters.getRaUsageLimitsPerInstant().forEach((instantName, raUsageLimits)
+            -> crac.newRaUsageLimits(instantName)
+            .withMaxRa(raUsageLimits.getMaxRa())
+            .withMaxTso(raUsageLimits.getMaxTso())
+            .withMaxRaPerTso(raUsageLimits.getMaxRaPerTso())
+            .withMaxPstPerTso(raUsageLimits.getMaxPstPerTso())
+            .withMaxTopoPerTso(raUsageLimits.getMaxTopoPerTso())
+            .add());
+    }
```

Comment:
put this in a new common class in crac-creation-util, since it is used by all CRAC creators

Reply:
I realize that I named it RaUsageLimitsAdder as it's the only thing it does but it's also the name of an Interface in crac api package. 
Perhaps it should be renamed otherwise to avoid misunderstandings.